client/components/app.jsx

```diff
@@ -8,10 +8,13 @@ class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      grades: []
+      grades: [],
+      view: 'add'
```

Comment:
Your state should also house which grade is being edited (at least the id). You can set it to `null` initially.